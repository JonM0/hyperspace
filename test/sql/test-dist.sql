select '(1,1,1,1)'::fourd <-> '(1,1,1,1)'::fourd = 0;
select 0 = '(1,1,1,1)'::fourd <-> '(1,1,1,1)'::fourd;