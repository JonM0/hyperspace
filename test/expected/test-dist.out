select '(1,1,1,1)'::fourd <-> '(1,1,1,1)'::fourd = 0;
 ?column? 
----------
 t
(1 row)

select 0 = '(1,1,1,1)'::fourd <-> '(1,1,1,1)'::fourd;
 ?column? 
----------
 t
(1 row)

